provision/acc_provision/flavors.yaml

```diff
@@ -381,6 +383,89 @@ flavors:
         use_opflex_server_volume: True
         use_kubeconfig_volume: True
         use_cnideploy_initcontainer: True
+        host_agent_cni_bin_path: /var/lib
+        host_agent_cni_conf_path: /etc/kubernetes
+      aci_config:
+        contracts:
+          - name: allow-all-out
+            underlay: True
+            filter: match-all
+            provided: ["inet-ext"]
+            consumed: ["ul-nodes", "ul-boot"]
+          - name: allow-boot
+            underlay: True
+            filter: match-boot
+            provided: ["ul-boot"]
+            consumed: ["inet-ext"]
+          - name: allow-cidr
+            underlay: True
+            filter: match-oshift
+            provided: ["ul-boot", "ul-nodes"]
+            consumed: ["cidr-ext"]
+          - name: allow-kube
+            underlay: False
+            filter: match-all
+            provided: ["aci-containers-default", "aci-containers-nodes", "aci-containers-system"]
```

Comment:
The "aci-containers-" prefix will work only when the "use_legacy_kube_naming_convention" field is not set. 
Is that an assumption here, not never have the older naming convention?

For OpenShift flavor, we use this: https://github.com/noironetworks/acc-provision/blob/5fde656c560c12c6a311e1d1d3379a5230e49bd3/provision/acc_provision/flavors.yaml#L121
And add the prefix based on the naming conventon.

Reply:
Because this is a new flavor, my thought is we don't need to worry about legacy.

Reply:
Can we add a check that doesn't allow the field to be set in this flavor then?

Reply:
sure.